From ad82706e5c66bd1d7dc5a29e7ba782206e1a904e Mon Sep 17 00:00:00 2001
From: Dave Gerlach <d-gerlach@ti.com>
Date: Wed, 22 Jul 2015 16:01:37 -0500
Subject: [PATCH 12/25] ARM: dts: dra74x: Remove non-common cpu0
 operating-points

Remove all OPPs that are not common to all dra74x variants.

Signed-off-by: Dave Gerlach <d-gerlach@ti.com>
---
 arch/arm/boot/dts/dra74x.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/dra74x.dtsi b/arch/arm/boot/dts/dra74x.dtsi
index 8bcc47d..3c1c942 100644
--- a/arch/arm/boot/dts/dra74x.dtsi
+++ b/arch/arm/boot/dts/dra74x.dtsi
@@ -24,7 +24,6 @@
 			operating-points = <
 				/* kHz    uV */
 				1000000	1060000
-				1176000	1160000
 				>;
 
 			clocks = <&dpll_mpu_ck>;
-- 
2.7.0.rc3

